mysql-test/include/have_outfile.inc

```diff
@@ -1,5 +1,5 @@
 let $outfile= `SELECT LOAD_FILE(CONCAT(@tmpdir, '/outfile.test')) LIKE 'Outfile OK%'`;
-remove_file $MYSQLTEST_VARDIR/tmp/outfile.test;
+remove_file $MYSQL_TMP_DIR/outfile.test;
```

Comment:
This change causes main.outfile to fail since that test creates the outfile in $MYSQLTEST_VARDIR/tmp/outfile.test. Maybe fsync test should change to use $MYSQLTEST_VARDIR/tmp instead?

Reply:
We changed this as the MYSQL_TMP_DIR is safer in parallel test runs (see above comments).

I could update main.outfile to use TMP_DIR instead?

Reply:
Yes, that would be fine too. Thanks!

Reply:
I just checked on the $MYSQLTEST_VARDIR variable. In a parallel test run, the path is:

/data/users/herman/rocks-mysql/8.0/_build-8.0-Debug/mysql-test/var/1/

So I think it already includes the parallel worker id in the pathname.